config.go

```diff
@@ -136,6 +136,12 @@ type autoPilotConfig struct {
 
 type torConfig struct {
 	Socks           string `long:"socks" description:"The port that Tor's exposed SOCKS5 proxy is listening on. Using Tor allows outbound-only connections (listening will be disabled) -- NOTE port must be between 1024 and 65535"`
+	Control         string `long:"control" description:"The port that Tor's ControlPort is listening on -- NOTE port must be between 1024 and 65535"`
```

Comment:
The docs above (and possibly semantics) will need to be modified in order to allow outbound connections to other onion service, yet still also listen with an onion service.